data/EventRow.css

```diff
@@ -1,10 +1,12 @@
 .event {
-    background: alpha(@colorAccent, 0.15);
+    background: alpha(@eventcolorAccent, 0.15);
```

Comment:
Since we add the provider to the widget and not the screen, this is not necessary